docs/pages/careers.tsx

```diff
@@ -36,22 +36,22 @@ function Role(props: RoleProps) {
   if (props.url) {
     return (
       <Box
-        component="div"
         sx={{
+          py: 1,
           display: 'flex',
+          flexDirection: { xs: 'column', lg: 'row' },
           justifyContent: 'space-between',
           alignItems: 'start',
-          flexDirection: { xs: 'column', lg: 'row' },
         }}
       >
-        <span>
-          <Typography variant="body1" color="text.primary" fontWeight="semiBold" gutterBottom>
+        <div>
+          <Typography variant="body1" color="text.primary" fontWeight="medium" gutterBottom>
             {props.title}
           </Typography>
-          <Typography component="p" color="text.secondary" sx={{ maxWidth: 700 }}>
+          <Typography variant="body2" color="text.secondary" sx={{ maxWidth: 650 }}>
```

Comment:
Great improvement on the copy! Just one thing, I'd make all the max-widths as 50% since it's what the grid is using on 'Perks & benefits'. What do you think?

| Current 650px | 50% |
|--------|--------|
| ![image](https://github.com/mui/material-ui/assets/37222944/d0648b0a-5819-4583-8203-d1fccd125c55) | ![image](https://github.com/mui/material-ui/assets/37222944/5deab5a4-9aab-4c22-9951-b8347e76b476) |

Reply:
Good call — pushed an overall refinement commit there taking this and other stuff.